### ISSUE TITLE
rule `comment-content` too strict

### ISSUE THREAD

**Issue by thernstig** [sindresorhus/eslint-plugin-unicorn#3253]
This fails on https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v66.0.0/docs/rules/comment-content.md:

```js
// TODO: Add https://github.com/ota-meshi/eslint-plugin-jsonc/
// when it supports @eslint/json
// HTML linting
```

It complains on `@eslint/json` and wants to change it to `@ESLint/JSON`. Which is obviously not correct since `@eslint/json` is an identifier.

Another example:

```js
// Run @svgr/webpack for .svg imports
```

It complains on `.svg` and changes it into `.SVG`.

Another example:

```js
    // {
    //   url: 'https://example.com',
    //   lastModified: new Date(),
    //   changeFrequency: 'monthly',
    //   priority: 0.7,
    // },
```

`url` -> `URL`.

I think this is a bad rule overall. It is impossible for it to know things like this.

**Comment by fregante** [comment]
Linking:

- https://github.com/sindresorhus/eslint-plugin-unicorn/pull/3104#issuecomment-4699446150
- https://github.com/sindresorhus/eslint-plugin-unicorn/pull/3236

**Comment by fregante** [comment]
> It is impossible for it to know things like this.

I agree, especially because it affects comments, which are difficult/annoying to consequently `eslint-disable`. 

There's actually another rule just like this that affects strings and had to be disabled for the same reason (although it was **more** problematic because it was _strings_ rather than _comments_)

- https://github.com/sindresorhus/eslint-plugin-unicorn/issues/630

**Comment by andreww2012** [comment]
https://github.com/antfu/case-police this plugin is not bad in terms of overlapping with this rule

**Comment by sindresorhus** [comment]
I will remove it from the `recommended` preset in the next update.
